Sources/CoreMetrics/Metrics.swift

```diff
@@ -221,18 +231,23 @@ public protocol TimerHandler: AnyObject {
 
 /// A timer collects observations within a time window (usually things like request durations) and provides aggregated information about the data sample,
 /// for example, min, max and various quantiles. It is similar to a `Recorder` but specialized for values that represent durations.
-/// This is the user facing Timer API. Its behavior depends on the `TimerHandler` implementation.
+///
+/// This is the user facing Timer API.
+///
+/// Its behavior depends on the `TimerHandler` implementation.
 public class Timer {
     @usableFromInline
     var handler: TimerHandler
     public let label: String
     public let dimensions: [(String, String)]
 
-    /// Create a new `Timer`.
+    /// Alternative way to create a new `Timer`, while providing an explicit `TimerHandler`.
```

Comment:
Maybe just

    /// Create a new `Timer` with an explicit `TimerHandler`.

Reply:
Maybe, though trying to stress the "this is not the api you're looking for" part by using the `Alternative...` spelling 🤔

Reply:
But not everyone reads or notices the function comments, either during autocompletion or by option-clicking the name.

Reply:
This changes the comment from one wording to another, so not sure what "people dont read comments" aims to highlight here; What's your suggestion?